### PR TITLE
Sync folder items

### PR DESCRIPTION
**Pull request by mhahnenberg**

### CODE REVIEW

File: exchangelib/folders.py [hunk 1]
```diff
@@ -455,6 +456,9 @@ def find_items(self, q, shape=IdOnly, depth=SHALLOW, additional_fields=None, ord
                     item.folder = self
                     yield item
 
+    def sync_items(self, shape, sync_state=None, ignore=None, max_changes=100):
```
Comment (emorikawa) [inline exchangelib/folders.py]: Maybe `sync_folder_items` to match tag?

File: exchangelib/services.py [hunk 2]
```diff
@@ -1069,6 +1069,105 @@ def get_payload(self, folders):
...
+
+        max_changes_elem = create_element('m:MaxChangesReturned')
+        max_changes_elem.text = str(max_changes)
+        sync_folder_items.append(max_changes_elem)
+        return sync_folder_items
+
+    def _get_elements_in_response(self, response):
+        result = super(SyncFolderItems, self)._get_elements_in_response(response)
+        for msg in response:
+            sync_state = self._get_element_container(message=msg, name='{%s}SyncState' % MNS)
```
Comment (emorikawa) [inline exchangelib/services.py:1168]: Mixed string formatting here?

**Review by emorikawa — APPROVED** [review]

File: exchangelib/services.py (hunk 2 above)
Comment (mhahnenberg) [reply exchangelib/services.py:1168]: Nope, same as the other diff.

File: exchangelib/folders.py (hunk 1 above)
Comment (mhahnenberg) [reply exchangelib/folders.py]: Done

**Pull request merged**